rented-properties/src/main/java/org/egov/cpt/web/controllers/PropertyController.java

```diff
@@ -112,4 +116,14 @@ public String create() {
 		return new ResponseEntity<>(response, HttpStatus.OK);
 	}
 
+	@PostMapping("/_accountstatementxlsx")
+	public ResponseEntity<String> generateAccountStatementExcel(
+			@Valid @RequestBody AccountStatementRequest request) {
+		/* Set current date in a toDate if it is null */
+		request.getCriteria().setToDate(
+				request.getCriteria().getToDate() == null ? new Date().getTime() : request.getCriteria().getToDate());
+		AccountStatementCriteria accountStatementCriteria = request.getCriteria();
+		String response = accountStatementExcelGeneration.generageAccountStatementExcel(accountStatementCriteria, request.getRequestInfo());

```

Comment:
We should be sending a json response.

Reply:
now returning a json response in the form of list